### PR TITLE
Bump skopeo image to v1.3.0 and buildah image to v1.18.0

### PR DESCRIPTION
**Pull request by barthy1**
# Changes

`quay.io/buildah/stable:v1.18.0` and `quay.io/skopeo/stable:v1.3.0` are multi-arch images, bumping images to these versions for buildah and skopeo-copy will allow to run the tasks on different architectures.
The update is tested for `linux/amd64`, `linux/s390x` and `linux/ppc64le` platforms. Both image versions have also `linux/arm64` support, so this platform is also mentioned.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Follows the [authoring recommendations](https://github.com/tektoncd/catalog/blob/main/recommendations.md)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

File: task/skopeo-copy/0.1/skopeo-copy.yaml [hunk 1]
```diff
@@ -40,7 +41,7 @@ spec:
       default: "true"
   steps:
     - name: skopeo-copy
-      image: quay.io/skopeo/stable:v1.1.1
+      image: quay.io/skopeo/stable:v1.2.0@sha256:6bbf47ed612e7c28c449c57d579565e3defdffc80b58c906186e180a851742e4
```
Comment (vinamra28) [inline task/skopeo-copy/0.1/skopeo-copy.yaml:44]: hey @barthy1, I see that v1.3.0 of skopeo has arm support as well so can't we use that tagged image here?

File: task/buildah/0.2/buildah.yaml [hunk 2]
```diff
@@ -25,7 +26,7 @@ spec:
     description: Reference of the image buildah will produce.
   - name: BUILDER_IMAGE
     description: The location of the buildah builder image.
-    default: quay.io/buildah/stable:v1.17.0
+    default: quay.io/buildah/stable:v1.18.0@sha256:120f0bf114bf02c14db672c5940615db220fb4be7a8a2e73e11fb48ff7f510de
```
Comment (vinamra28) [inline task/buildah/0.2/buildah.yaml:29]: can you please remove the digest from the image as whenever there is a push on this tag the previous digest becomes in-accessible. This is mainly the scenario in quay and might break the task in future :sweat_smile:

Comment (barthy1) [reply task/buildah/0.2/buildah.yaml:29]: Sure, sorry, I didn't know. I just followed hints after catlin check :)

File: task/skopeo-copy/0.1/skopeo-copy.yaml (hunk 1 above)
Comment (barthy1) [reply task/skopeo-copy/0.1/skopeo-copy.yaml:44]: sure. However I don't have arm hardware to run the tests.

**Review by vdemeester — APPROVED** [review]

**Pull request merged**